packager/accept/accept_test.go

```diff
@@ -11,16 +11,17 @@ func TestCanSatisfy(t *testing.T) {
 	assert.False(t, CanSatisfy("*/*"))
 	assert.False(t, CanSatisfy("image/jpeg;v=b3"))
 	assert.False(t, CanSatisfy(`application/signed-exchange;v=b2`))
-	// This is a bug that will be triggered when a UA starts supporting multiple SXG versions:
-	assert.False(t, CanSatisfy(`application/signed-exchange;x="a,b";v="b3"`))
+	assert.False(t, CanSatisfy(`application/signed-exchange;v="b1,b2"`))
+	assert.False(t, CanSatisfy(`application/signed-exchange;x="y,application/signed-exchange;v=b3,z";v=b1`))
```

Comment:
Add some tests including `\\"` (escaped backslash followed by unescaped quote) and `\\\"` (escaped backslash followed by escaped quote), to guard against `tokenize` being replaced by something simpler but wrong.

Reply:
Could issue #293 be used for this request?

Reply:
It's not addressed by the CL in question, but I'm okay moving this into that issue.